static/styles/zulip.css

```diff
@@ -1373,7 +1390,7 @@ td.pointer {
        private messages, mentions, and unread messages */
 
     .message-header-contents {
-        background-color: hsla(192, 19%, 75%, 0.2);
+        background-color: hsl(104.2, 100%, 92.5%);
```

Comment:
Temporary color -- the actual recipient header is being worked on in a separate PR.

Reply:
@alya we actually probably shouldn't merge this PR until we have a color for the recipient header, how's the progress on that going?

Reply:
or maybe this is fine?

<img width="651" alt="image" src="https://user-images.githubusercontent.com/5634097/217739753-ed802528-67e2-418a-9ca4-d2f460293cac.png">

Reply:
@amanagr Would know best about the recipient header colors PR status. But in any case the next step is to test on CZO, which I think we can do without getting the header bars updated.